Makefile

```diff
@@ -669,6 +672,14 @@ test-capd-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests
 kind-cluster: ## Create a new kind cluster designed for development with Tilt
 	hack/kind-install-for-capd.sh
 
+.PHONY: kind-reset
+kind-reset: ## Destroys the kind cluster with the name $CAPI_KIND_CLUSTER_NAME.
+	kind delete cluster --name="$(CAPI_KIND_CLUSTER_NAME)" || true #
```

Comment:
Out of interest: What is the purpose at the `#` at the end of the command.

Reply:
I think that was a typo